tests/PBurggraf/LicensePlate/GermanyDetectorTest.php

```diff
@@ -331,6 +332,8 @@ public function testValidity($plate, $validity, $type)
             $result = $validationResult->isValid();
             $this->assertEquals($validity, $result, sprintf('Tested plate: \'%s\', should be %s', $plate, (bool)$validity));
         }
+
+        $this->assertGreaterThanOrEqual(0, count($validationResults));
```

Comment:
On some test cases, `$validationResults` will be empty array and it will present `This test did not perform any assertions` during PHPUnit tests running.

To fix this issue, it should add this line to let all results can assert length is greater than or equal to `0`.